UpdateManagement-TurnOnVms.ps1

```diff
@@ -141,15 +141,15 @@ $vmIds | ForEach-Object {
     Write-Output ("Subscription Id: " + $subscriptionId)
     $mute = Select-AzureRmSubscription -Subscription $subscriptionId
 
-    $vm = Get-AzureRmVM -ResourceGroupName $rg -Name $name -Status 
+    $vm = Get-AzureRmVM -ResourceGroupName $rg -Name $name -Status -DefaultProfile $mute 
 
     #Query the state of the VM to see if it's already running or if it's already started
     $state = ($vm.Statuses[1].DisplayStatus -split " ")[1]
     if($state -in $startableStates) {
         Write-Output "Starting '$($name)' ..."
         #Store the VM we started so we remember to shut it down later
         $updatedMachines += $vmId
-        $newJob = Start-ThreadJob -ScriptBlock { param($resource, $vmname) Start-AzureRmVM -ResourceGroupName $resource -Name $vmname} -ArgumentList $rg,$name
+        $newJob = Start-ThreadJob -ScriptBlock { param($resource, $vmname, $sub) $context = Select-AzureRmSubscription -Subscription $sub; Start-AzureRmVM -ResourceGroupName $resource -Name $vmname -DefaultProfile $context} -ArgumentList $rg,$name,$subscriptionId
```

Comment:
is there any other backend pr that is supposed to handle this logic ?

Reply:
no, this is just a sample runbook uploaded in gallery so that users can use it. We got a CRI for tis for multiple sub VMs and we are fixing it. No change needed in nay other place.

Reply:
as a customer I can verify this works, I also adapted the same change for the power off script